dmutils/content_loader.py

```diff
@@ -464,18 +483,33 @@ def __init__(self, question, service_data):
         self.questions = question.questions
         if self.questions:
             self.questions = [q.summary(service_data) for q in self.questions]
+        self.fields = question.fields
+
+    def _default_for_field(self, field_key):
+        return self.get('field_defaults', {}).get(field_key)
 
     @property
     def value(self):
         if self.questions:
             return self.questions
+        if self.type == "pricing":
+            return format_price(
+                self._service_data.get(self.fields.get('minimum_price')),
+                self._service_data.get(self.fields.get('maximum_price')),
+                self._service_data.get(self.fields.get('price_unit'), self._default_for_field('price_unit')),
+                self._service_data.get(self.fields.get('price_interval'), self._default_for_field('price_interval')),
+            )
         return self._service_data.get(self.id, '')
 
     @property
     def answer_required(self):
         if self.questions:
             return any(question.answer_required for question in self.questions)
-        if self.value in ['', [], None]:
+        elif self.fields:
+            return any(
+                self._service_data.get(field_name) in ['', [], None]
```

Comment:
If `format_price` returned an empty string when none of the fields are set we could just check `self.value` here like we do for normal questions